src/services/AuthService.ts

```diff
@@ -256,7 +256,7 @@ const AuthService = {
       }
       const userStore = useUserStore();
       userStore.updateCurrentUser(authenticatedUser);
-      await userStore.getAllFromUserDatabase();
+      if (userStore.$state.currentUser?.id !== authenticatedUser.id) await userStore.getAllFromUserDatabase();
```

Comment:
why do you have to do $state on userstore? Isn't currentUser directly accessible at the top level?